docs/_pages/releases.md

```diff
@@ -9,8 +9,12 @@ sidebar:
 
 ## Unreleased
 
+**NOTE**
+From this release on `MethodInfoSelector` is now including static methods!
```

Comment:
Please phrase it as a fix and keep it functional. Nobody knows what `MethodInfoSelector` is. What does it mean to a consumer of this library? And also please add the same PR information.

Reply:
OK, have a look now.